yt/frontends/fits/data_structures.py

```diff
@@ -540,7 +540,11 @@ def _determine_axes(self):
 
     @classmethod
     def _is_valid(cls, filename, *args, **kwargs):
-        fileh = check_fits_valid(filename)
+        try:
+            fileh = check_fits_valid(filename)
+        except Exception:
+            return False
+
```

Comment:
So do we expect this to crash if the file is non-existent or FITS can't be imported or something?